client/src/components/account/AuthContainer.js

```diff
@@ -12,11 +13,53 @@ class AuthContainer extends Component {
       username: '',
       password: '',
       passwordCheck: '',
+      newPass: '',
+      newPassCheck: '',
       errorMessage: ''
     }
 
     this.attemptLogin = this.attemptLogin.bind(this);
     this.attemptRegister = this.attemptRegister.bind(this);
+    this.attemptPassChange = this.attemptPassChange.bind(this);
+  }
+
+  attemptPassChange(e) {
+    e.preventDefault()
+    if (this.state.username && this.state.password && this.state.newPass && this.state.newPassCheck) {
+      if (this.state.newPass === this.state.newPassCheck) {
+        fetch('/changepass/', {
+          method: 'PUT',
+          body: JSON.stringify({ username: this.state.username, password: this.state.password, newPass: this.state.newPass}),
+          headers: new Headers({ 'Content-type': 'application/json' })
```

Comment:
see TutorialContainer.js getTutorial() method to see how jwt's work. The user must have a jwt to change a password so check with `if (jwt)`